modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/utils/Alerting.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2014-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.snowplow.rdbloader.utils
+
+
+import blobstore.s3.{S3Path, S3Store}
+import cats.{Applicative, Parallel}
+import cats.implicits._
+import cats.effect.{Async, ConcurrentEffect, ContextShift, Sync, Timer}
+import com.snowplowanalytics.snowplow.rdbloader.LoaderAction
+import com.snowplowanalytics.snowplow.rdbloader.common.S3
+import com.snowplowanalytics.snowplow.rdbloader.common.config.StorageTarget
+import com.snowplowanalytics.snowplow.rdbloader.config.CliConfig
+import com.snowplowanalytics.snowplow.rdbloader.db.Statement._
+import com.snowplowanalytics.snowplow.rdbloader.dsl.Monitoring.AlertPayload
+import com.snowplowanalytics.snowplow.rdbloader.dsl.{AWS, Environment, JDBC, Logging, Monitoring}
+import com.snowplowanalytics.snowplow.rdbloader.dsl.alerts.Alerter.createAlertPayload
+import doobie.util.Read
+import fs2.Stream
+import fs2.text.utf8Encode
+
+
+object Alerting {
+
+  implicit val folderRead: Read[S3.Folder] = Read[String].map(S3.Folder.coerce)
+
+  def putAlertScanToS3[F[_]: AWS: Sync](s3Store: S3Store[F], shredderOutput: S3.Folder): F[S3.Key] = {
```

Comment:
The whole point of `AWS` capability is to abstract away from things like `S3Store` and not passing it around.